### PR TITLE
Add griefer code

### PR DESCRIPTION
**Pull request by crisleiva**
- add griefers code

### CODE REVIEW

**Comment by crisleiva** [comment]
Be gentle

File: main.go [hunk 1]
```diff
@@ -43,13 +49,49 @@ func onReady(s *discordgo.Session, event *discordgo.Ready) {
 	log.Printf("Logged in as %s\n", event.User.String())
 }
 
+func addGriefer(s *discordgo.Session, m *discordgo.MessageCreate) {
```
Comment (b1naryth1ef) [inline main.go]: We're duplicating a bit of logic here for no reason so it'd be better to consolidate this event handler to a single function (the `messageCreate` we already have), and then split the logic out into sub-functions (`handleGriefer(...)`, `handleHello(...)`, etc).

File: main.go [hunk 2]
```diff
@@ -43,13 +49,49 @@ func onReady(s *discordgo.Session, event *discordgo.Ready) {
...
+	if m.Author.ID == s.State.User.ID {
+		return
+	}
+
+	parts := strings.Split(m.Content, " ")
+
+	if parts[0] == "!grief" {
```
Comment (b1naryth1ef) [inline main.go:63]: I am quite sure we can write this with less nested if-statements.

File: main.go [hunk 3]
```diff
@@ -60,9 +102,23 @@ func messageCreate(s *discordgo.Session, m *discordgo.MessageCreate) {
...
+	}
+
+	for _, griefee := range griefers {
+		if m.VoiceState.UserID == griefee {
+			s.GuildMemberMove(m.GuildID, griefee, &channelId)
+		}
+	}
+}
+
+// 209404729225248769
```
Comment (b1naryth1ef) [inline main.go]: delete.

**Review by b1naryth1ef — APPROVED** [review]
😹

**Review by braydencw1 — APPROVED** [review]

**Pull request merged**